app/views/dashboard/locations/_form.html.erb

```diff
@@ -151,7 +151,7 @@
                     </div>
 
                     <div class="" data-opening-time-target="open24hBlock" >
-                      <%= fot.hidden_field :open_24h, value: false %>
```

Comment:
Is this deletion intentional?

Reply:
Yes! This was leading to the `open_24` field included twice and to an error when trying to save the location. I don't even remember why we added this, because it works withou it. Maybe it's also because something has changed with Rails 7.1?